tasks/curator.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Add Curator apt repo.
+  apt_repository:
+    repo: "deb http://packages.elastic.co/curator/5/debian stable main"
+    state: present
+
+- name: Install Curator.
+  apt:
+    name: elasticsearch-curator
+    state: latest
+    update_cache: yes
+
+- name: Create Curator configuration directory.
+  file:
+    path: /etc/curator
+    state: directory
+    mode: "0755"
+    owner: root
+    group: root
+
+- name: Copy Curator configuration.
+  file:
```

Comment:
Since curator is using yaml, I recommend you shove all the default parameters into ansible vars and then use the `copy` module along with the `content` field to feed this file. See https://github.com/freedomofpress/ansible-role-kibana/blob/master/tasks/configure.yml#L4 as an example of this